go-controller/pkg/ovn/master.go

```diff
@@ -174,6 +176,62 @@ func (oc *Controller) SetupMaster(masterNodeName string) error {
 	return nil
 }
 
+func parseNodeManagementPortMacAddr(node *kapi.Node) (string, error) {
+	macAddress, ok := node.Annotations[OvnNodeManagementPortMacAddress]
+	if !ok {
+		logrus.Errorf("macAddress annotation not found for node %q ", node.Name)
+		return "", nil
+	}
+
+	_, err := net.ParseMAC(macAddress)
+	if err != nil {
+		return "", fmt.Errorf("Error %v in parsing node macAddress %v", err, macAddress)
+	}
+
+	return macAddress, nil
+}
+
+func (oc *Controller) syncNodeManagementPort(node *kapi.Node) error {
+
+	macAddress, err := parseNodeManagementPortMacAddr(node)
+	if err != nil {
+		return err
+	}
+
+	if macAddress == "" {
+		// When macAddress was removed, delete the switch port
+		stdout, stderr, err := util.RunOVNNbctl("--", "--may-exist", "lsp-del", node.Name, "k8s-"+node.Name)
+		if err != nil {
+			logrus.Errorf("Failed to delete logical port to switch, stdout: %q, stderr: %q, error: %v", stdout, stderr, err)
+		}
+
+		return nil
+	}
+
+	subnet, err := parseNodeHostSubnet(node)
+	if err != nil {
+		return err
+	}
+	if subnet == nil {
+		return fmt.Errorf("error obtaining subnet for macAddress %v of node %s", macAddress, node.Name)
+	}
+
+	_, portIP, _, err := util.GetNodeWellKnownAddresses(subnet)
+	if err != nil {
+		logrus.Errorf("Failed to obtain management port IP address, error: %v", err)
+		return err
+	}
+
+	// Create this node's management logical port on the node switch
+	stdout, stderr, err := util.RunOVNNbctl("--", "--may-exist", "lsp-add", node.Name, "k8s-"+node.Name, "--", "lsp-set-addresses", "k8s-"+node.Name, macAddress+" "+portIP)
```

Comment:
Also, you are missing to remove the `exclude_ips`. Before we had
```
	stdout, stderr, err = util.RunOVNNbctl("--", "--may-exist", "lsp-add", nodeName, "k8s-"+nodeName,
		"--", "lsp-set-addresses", "k8s-"+nodeName, macAddress+" "+portIP,
		"--", "--if-exists", "remove", "logical_switch", nodeName, "other-config", "exclude_ips")
```
@L226 you are missing to remove the `exclude_ips`